test/langtools/tools/javap/T4975569.java

```diff
@@ -27,6 +27,7 @@
  * @summary javap doesn't print new flag bits
  * @modules jdk.jdeps/com.sun.tools.javap
  * @modules java.base/jdk.internal.misc
+ * @requires !java.enablePreview
```

Comment:
Why is  `@requires !java.enablePreview` needed here? The test's `id` value is `NoPreview` so that might
indicate that the test doesn't support `--enable-preview` bits, but I had to expand the review's context
to see that `id` value.

Reply:
Added a comment.